tools/mo/openvino/tools/mo/utils/cli_parser.py

```diff
@@ -996,8 +996,12 @@ def parse_layouts_by_destination(s: str, parsed: dict, dest: str = None) -> None
             elif m2:
                 found_g = m2.groups()
             else:
-                raise Error("More then one layout provided for --{}layout without providing name.".format(
-                    dest + '_' if dest else ''))
+                raise Error("Invalid usage of --{}layout parameter. Please use following syntax for each tensor or "
+                            "operation name:\n"
+                            "  name(nchw)\n"
+                            "  name(nhwc->[n,h,w,c])\n"
+                            "  name[n,c,h,w]\n"
+                            "  name[n,h,w,c]->[n,c,h,w]".format(dest + '_' if dest else ''))
```

Comment:
Please double-check if it is correct to recommend `name[n,h,w,c]->[n,c,h,w]` for `source_layout` or `target_layout`.

The text looks good!

Reply:
You are right, `->` is not applicable for `source_layout` or `target_layout`. Corrected.